test/BaseFixture.sol

```diff
@@ -23,6 +23,7 @@ contract BaseFixture is Test {
 
     uint256 constant EURE_TO_MINT = 1_000e18;
     uint128 constant MIN_EURE_ALLOWANCE = 200e18;
+    uint256 constant EURE_BUFFER = 50e18;
```

Comment:
im thinking it makes sense to simply make the buffer equal to the daily limit. that way technically there wont be a need for any action more than once a day

Reply:
wdyt? so lower bound = daily limit, upper bound = daily limit * 2

Reply:
i will prefer to allow giving user flexibility on determining what they think is appropriate for them as "buffer", but the simply daily x 2 can also work as well for poc